docs/reference/command-arguments.md

```diff
@@ -16,6 +16,9 @@ Every element in the _arguments_ array is a map with the following fields:
 * **name:** the argument's name, always present.
   The name of an argument is given for identification purposes alone.
   It isn't displayed during the command's syntax rendering.
```

Comment:
Is this correct? Isn't name the default display?

Reply:
no, display is the default display (it's just that if "display" is not present in the json, COMMANDS DOCS will use "name" for "display")

bottom, line, display is or display, name is for identification

Reply:
so don't we still need to clear that out?
i.e. that the `name` should be used as display only if display_text is missing?

also, maybe we wanna mention something about the uniqueness of `name`?

Reply:
"display_text" is never missing in COMMAND DOCS (except for block/oneof, in which case there's nothing to display)

and yes, i'll add a few words on the uniqueness of `name`

Reply:
perhaps a piece of code is worth more than a thousand comments:
```
addReplyBulkCString(c, "display_text");
addReplyBulkCString(c, args[j].display_text ? args[j].display_text : args[j].name);
 ```

Reply:
ohh, i managed to forget we closed that loop in redis.